companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -1217,7 +1217,7 @@ unsigned long OpenTxEepromInterface::loadBackup(RadioData &radioData, const uint
 
 QString OpenTxFirmware::getReleaseNotesUrl()
 {
-  return QString("%1/downloads").arg(EDGETX_HOME_PAGE_URL);
+  return QString( tr("%1/downloads")).arg(EDGETX_HOME_PAGE_URL);
```

Comment:
Er... is it valid to be translating this? Isn't it constructing the release notes URL path? i.e. `EDGETX_HOME_PAGE_URL/downloads` (which isn't actually used/valid anyway - it directly goes to QString link = "https://github.com/EdgeTX/edgetx/releases";

Reply:
yrk,. was supposed to change the local catalog at $USER/EdgeTx/downloads.
I do a rollback, but the rollback will not give **"https://github.com/EdgeTX/edgetx/releases"** but (in constant.h) **"https://edgetx.org"**
I'll change to ..edgetx/releases in the translation files , ok ?

Reply:
no, will not work, damn, I restore the translation files but danish one, and change constant.h to the new one, ok ?

Reply:
You should be able to just undo all the changes to everything but Danish translation file... if needed I can probably roll that back, just me know if you give up ;)

The change you made here `opentxinterface.cpp` was the addition of the `tr()` - I was just pointing that string isn't even used, even it it was valid to be translated...
![image](https://user-images.githubusercontent.com/5500713/196639753-b6548915-a8d7-4698-b509-d0160acd8a22.png)

Reply:
I try to make the rollback, something to learn :-)